doc/website/concepts/architecture.md

```diff
@@ -1 +1,65 @@
 # Architecture
+
+This article provides an overview of the Eclipse iceoryx architecture and explains the rationale for it.
+
+## Software layers
+
+The main packages which Eclipse iceoryx is composed of are depicted below.
+
+![Software layers](../images/iceoryx_software_layers_v2_0_0.svg)
+
+The next sections briefly describe the components and their libraries one by one.
+
+## Components and libraries
+
+The different libraries and their namespaces are depicted below.
+
+![Component diagram](../images/iceoryx_components_diagram_v2_0_0.svg)
+
+### iceoryx hoofs
+
+Handy Objects Optimized For Safety (hoofs) is a library and contains various building blocks like fixed size containers,
+concurrency classes and modern, next-gen C++ constructs from upcoming C++ standard releases.
+
+For more information about the components, refer to its [detailed description](../advanced/iceoryx_hoofs.md).
+
+### iceoryx posh
+
+The package `iceoryx_posh` (**PO**SIX **SH**ared memory) contains everything related to shared memory inter-process communication.
+
+#### Core library
+
+This section describes the namespaces of the core library.
+
+* `popo`: The namespace posh ports contains the user-API classes which are used to transfer data.
+* `capro`: The namespace canonical protocol implements parts of the [SoA](https://en.wikipedia.org/wiki/Service-oriented_architecture)
```

Comment:
Actually, capro is short for CAnonical PROtocol and implemented originally this pattern here: https://en.wikipedia.org/wiki/Canonical_protocol_pattern

Reply:
Good catch, thanks! Replaced the link.